sandbox/src/main/java/com/bumble/appyx/sandbox/client/interop/child/RibsChild.kt

```diff
@@ -1,11 +1,14 @@
 package com.bumble.appyx.sandbox.client.interop.child
 
 import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.Rib
 import com.badoo.ribs.core.modality.BuildParams
 import com.badoo.ribs.core.view.RibView
 
+interface RibsChild : Rib
```

Comment:
Is this needed?

Reply:
I added it to be able to create the `RibsChildBuilder`, so both children are created using builders and also to create [this test](https://github.com/bumble-tech/appyx/pull/65/files#diff-7a3789fc1583c7c46c8dfaa87fedab0b959dc4fea991639161f679c206e1f441R72) as an example

Reply:
It was not needed, I removed it in 4cec1addcaa4010a67d8fe2b6b262e947952fc5b and used `Rib` directly